src/sentry/models/organization.py

```diff
@@ -328,3 +330,25 @@ def send_delete_confirmation(self, audit_log_entry, countdown):
     def flag_has_changed(self, flag_name):
         "Returns ``True`` if ``flag`` has changed since initialization."
         return getattr(self.old_value('flags'), flag_name, None) != getattr(self.flags, flag_name)
+
+    def send_setup_2fa_emails(self):
+        from sentry import options
+        from sentry.utils.email import MessageBuilder
+
+        for member in self.member_set.all():
```

Comment:
Let's change this queryset a bit:

```python
users = User.objects.filter(
    is_active=True,
    sentry_orgmember_set__organization=self,
)
```

So this does a few things:

* We're querying directly for User instead of OrganizationMember, then dipping into member.user.
* Not all OrganizationMember rows are actually active. Some of them have `NULL` for `user_id` which would result in an `ObjectDoesNotExist` being raised when you accessed `member.user`. This is the case when there are outstanding pending user invites.
* Limits it to actually active user accounts, and not disabled ones.

Reply:
That's really good insight! Thanks! Can do.